### DISCUSSION TITLE
CRON (CLI) not change after php.ini change

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14807] [How to]
After changing the php.ini file to solve red line in attachment of CRON (CLI) column, the red lines not change their status.

Why?
![1234567](https://user-images.githubusercontent.com/85027316/121172203-bcca0c00-c857-11eb-8906-fb4180c082f5.JPG)

**Comment by waran70** [comment]
O.o
systemctl reload apache2

**Comment by espe808** [comment]
I have launched the apache2 command, but anyway it doesn't work

**Comment by mariuszkrzaczkowski** [comment]
check this
![image](https://user-images.githubusercontent.com/10198654/121185112-04579480-c866-11eb-8ffb-42ed0d5d7b91.png)

CLI usually does not need to be reloaded, it loads the current ini file each time it is run

delete the file app_data / cron.php
run cron (manually from CLI) and attach full configuration report

**Reply by espe808** [reply]
Hello @mariuszkrzaczkowski ,
thanks for your reply.

I have solved the issues as your suggestion, but now I can import only 1000 records (Accounts, Contacts) each time.
Why?

![fdsfsdfsdfsdfsdf](https://user-images.githubusercontent.com/85027316/121318535-9ebde380-c90b-11eb-8d89-9f4e4b9939de.png)

**Reply by mariuszkrzaczkowski** [reply]
Correct everything in red

` I can import only 1000 records (Accounts, Contacts) each time.`
create a new discussion

**Reply by espe808** [reply]
Ok. See here: https://github.com/YetiForceCompany/YetiForceCRM/discussions/14810
